internal/observability/middleware.go

```diff
@@ -160,6 +160,8 @@ func (m *Middleware) Call(ctx context.Context, req *transport.Request, out trans
 // HandleOneway implements middleware.OnewayInbound.
 func (m *Middleware) HandleOneway(ctx context.Context, req *transport.Request, h transport.OnewayHandler) error {
 	call := m.graph.begin(ctx, transport.Oneway, _directionInbound, req)
+	defer m.handlePanicForCall(call)
```

Comment:
Please add this to the `Handle` method too and add a test for its invocation. You can drop this line if you choose, since we effectively do not support oneway.